ocaml/xapi-cli-protocol/cli_protocol.ml

```diff
@@ -26,6 +26,13 @@ let minor = 2
     a totally different kind of server (eg a standard HTTP server) *)
 let prefix = "XenSource thin CLI protocol"
 
+(* Be careful to add/remove/modify the commands. The CLI interface is expected
+   to be extreme stable. Please keep following in mind when doing the changes:
+     1. backwards compatibility support. E.g. a very old CLI client may need to
+        be supported still,
```

Comment:
perhaps add a suggestion that the protocol minor number could be bumped for this purpose (which is available to the server as well) if you really have to.
But that hasn't ever been done, so it needs a lot of testing.